services/IoT/applications/libraries/java.md

```diff
@@ -350,15 +350,15 @@ The following code sample shows how applications can publish events as if they o
     myClient.publishEvent(deviceType, deviceId, "blink", event);
 ```
 
-### Publish events using HTTP(s)
+### Publish events by using HTTP
```

Comment:
do not put (s) in parenthesis. I've changed it to HTTP. If the (s) is important, then use "HTTP or HTTPS" every time (except for in line 379, when both HTTP and HTTPS are mentioned by name - leave those two references as is)